recipes/cath-tools/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -x -e
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="-I$PREFIX/include"
+export CPATH="${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export CXXFLAGS="-I${PREFIX}/include"
+
+
+#list programs
+CATH_PROGRAMS="build-test cath-assign-domains cath-cluster cath-map-clusters cath-refine-align cath-resolve-hits cath-score-align cath-ssap cath-superpose"
+
+#compile
+DCMAKE=""
+if [[ ${HOST} =~ .*darwin.* ]]; then
+	MACOSX_DEPLOYMENT_TARGET=10.9
+	DCMAKE+=" -DCMAKE_THREAD_LIBS_INIT=-lpthread -DCMAKE_HAVE_THREADS_LIBRARY=1"
+	DCMAKE+=" -DCMAKE_USE_PTHREADS_INIT=1"
+	export CXXFLAGS="${CXXFLAGS} -fvisibility=hidden -fvisibility-inlines-hidden"
+fi
+
+cmake -DGSL_LIBRARIES=${PREFIX}/include \
+			-DCMAKE_BUILD_WITH_INSTALL_NAME_DIR=ON \
+			-DCMAKE_INSTALL_RPATH="" \
+			-DCMAKE_BUILD_TYPE=Release \
+			-DCMAKE_INSTALL_PREFIX=$PREFIX \
+			-DBoost_DEBUG=ON \
```

Comment:
Is it necessary to set `Boost_DEBUG`? That sounds as if the resulting binary may be slower than necessary.

Reply:
removed along with -DCMAKE_INSTALL_RPATH="" that sounds useless